migrations/educandu-2021-11-18-01-add-document-cdn-resources.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line camelcase
+export default class Educandu_2021_11_18_01_add_document_cdn_resources {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async up() {
+    await this.db.collection('documentRevisions').updateMany({}, { $set: { cdnResources: [] } });
```

Comment:
Hmm I named these cdnResourceUrls, I'll rename to your variant.

Reply:
I think the tickets say `cdnResources` :thinking:

Reply:
Yep, my bad